Django 1.8 and python3 compatibilty

paxnovem opened the pull request:
I fixed some imports that changed with the update to django 1.8, but kept backwards compatibility by using a try catch statement with the old import. Instead of using `assertIn` the `response.content` I used the `assertContains` which is available back to 1.3 and solves issues with 1.8

tkaemming commented:
Thanks for the patch!

Was there a rationale from excluding Python 2.6 when testing Django 1.6 in the matrix? It appears to have been supported: https://docs.djangoproject.com/en/1.8/releases/1.6/

Also, the updated Travis configuration only is testing up to Django 1.7, since 1.8 is set as non-inclusive the upper bound on this version range: https://github.com/disqus/django-mailviews/pull/16/files#diff-354f30a63fb0907d4ad57269548329e3R17

Other than those couple things, I think that this is ready to land.

paxnovem commented:
Sorry about that django 1.6 should be tested against python 2.6, I fixed the travis file so 1.7 and 1.8 are excluded from python 2.6 and added django 1.8

tkaemming commented:
Thanks for the changes!

I'm going to release this as 0.6.4, since it shouldn't shouldn't break backwards compatibility.

I also improved the Travis configuration to have more readable version specifications, so hopefully any confusion around versions tested is avoided in the future: ba5269ffcda5d68f8340ae845ff041b670707ba2
